frontend/views/containers/chatroom/MessageInteractive.vue

```diff
@@ -73,14 +51,14 @@ const interactiveMessage = (proposal, baseOptions = {}) => {
     [PROPOSAL_INVITE_MEMBER]: {
       [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to add new members to the group.', options),
       [PROPOSAL_VARIANTS.EXPIRING]: L('Proposal from {from} to add new members is expiring.', options),
-      [PROPOSAL_VARIANTS.ACCEPTED]: L('Added members to this group: {to}', options),
+      [PROPOSAL_VARIANTS.ACCEPTED]: L('Added members to this group: {group}', options),
       [PROPOSAL_VARIANTS.REJECTED]: L('No members were added.'),
       [PROPOSAL_VARIANTS.EXPIRED]: L('No members were added.')
     },
     [PROPOSAL_REMOVE_MEMBER]: {
       [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to remove members from the group.', options),
       [PROPOSAL_VARIANTS.EXPIRING]: L('Proposal from {from} to remove members is expiring.', options),
-      [PROPOSAL_VARIANTS.ACCEPTED]: L('Left {title}', options),
+      [PROPOSAL_VARIANTS.ACCEPTED]: L('Left {member}', options),
```

Comment:
Shouldn't this say something like, `{member} was removed from the group.` ?

Reply:
```
[PROPOSAL_REMOVE_MEMBER]: {
      [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to remove {member} from the group.', options),
      [PROPOSAL_VARIANTS.EXPIRING]: L('Proposal from {from} to remove {member} is expiring.', options),
      [PROPOSAL_VARIANTS.ACCEPTED]: L('Proposal from {from} to remove {member} is accepted.', options),
      [PROPOSAL_VARIANTS.REJECTED]: L('Proposal from {from} to add {member} is rejected.', options),
      [PROPOSAL_VARIANTS.EXPIRED]: L('Proposal from {from} to add {member} is expired.', options)
}
```

Then, are you fine with these messages?

Reply:
Yes 👍